purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/offlineentitlements/OfflineEntitlementsWithInitialRequestsCompletedIntegrationTest.kt

```diff
@@ -1,315 +1,315 @@
-//
-//        ensureBlockFinishes { latch ->
-//            Purchases.sharedInstance.forceServerErrors = true
-//
-//            mockPurchaseResult()
-//            mockBillingAbstract.mockQueryProductDetails(queryProductDetailsSubsReturn = listOf(storeProduct))
-//            Purchases.sharedInstance.purchaseWith(
-//                PurchaseParams.Builder(activity, storeProduct).build(),
-//                onError = { error, _ ->
-//                    fail("Expected success but got error: $error")
-//                },
-//                onSuccess = { _, customerInfo ->
-//                    assertCustomerInfoHasExpectedPurchaseData(customerInfo)
-//                    assertAcknowledgePurchaseDidNotHappen()
-//
-//                    Purchases.resetSingleton()
-//                    mockActivePurchases(initialActivePurchases)
-//                    configureSdk(activity, forceServerErrors = false)
-//
-//                    assertAcknowledgePurchaseDidHappen()
-//
-//                    latch.countDown()
-//                }
-//            )
-//        }
-//    }
-//
-//    @Test
-//    fun recoversFromOfflineEntitlementsModeIfGetCustomerInfoSucceeds() {
-//        val storeProduct = StoreProductFactory.createGoogleStoreProduct()
-//
-//        ensureBlockFinishes { latch ->
-//            Purchases.sharedInstance.forceServerErrors = true
-//
-//            mockPurchaseResult()
-//            mockBillingAbstract.mockQueryProductDetails(queryProductDetailsSubsReturn = listOf(storeProduct))
-//            Purchases.sharedInstance.purchaseWith(
-//                PurchaseParams.Builder(activity, storeProduct).build(),
-//                onError = { error, _ ->
-//                    fail("Expected success but got error: $error")
-//                },
-//                onSuccess = { _, customerInfo ->
-//                    assertCustomerInfoHasExpectedPurchaseData(customerInfo)
-//                    assertAcknowledgePurchaseDidNotHappen()
-//
-//                    Purchases.sharedInstance.forceServerErrors = false
-//
-//                    Purchases.sharedInstance.getCustomerInfoWith(
-//                        CacheFetchPolicy.FETCH_CURRENT,
-//                        onError = {
-//                            fail("Expected success but got error: $it")
-//                        },
-//                        onSuccess = {
-//                            // This is a known limitation. Ideally we would sync unsynced purchases
-//                            // as soon as possible to avoid getting outdated info from the backend.
-//                            assertCustomerInfoDoesNotHavePurchaseData(it)
-//                            assertAcknowledgePurchaseDidNotHappen()
-//
-//                            latch.countDown()
-//                        }
-//                    )
-//                }
-//            )
-//        }
-//    }
-// }
+ package com.revenuecat.purchases.offlineentitlements
```

Comment:
There are no changes in this file, other than uncommenting the tests that were commented when offline entitlements was disabled. Ideally, we should conditionally run these tests depending on the flag, but commenting them out was easier for now.